### ISSUE TITLE
1/30: train_loss = nan, val_loss = nan

### ISSUE THREAD

**Issue by SheldonLiu0412** [Abhijit-Bhumireddy99/RUL_Prediction#5]
When I follow the instructions and reach the fourth step, and encounter this situation, is this a case of gradient explosion?
